internal/concierge/apiserver/apiserver.go

```diff
@@ -56,14 +57,13 @@ type CompletedConfig struct {
 
 // Complete fills in any fields not set that are required to have valid data. It's mutating the receiver.
 func (c *Config) Complete() CompletedConfig {
+	// Be sure to set this before calling c.GenericConfig.Complete()
+	c.GenericConfig.EffectiveVersion = utilversion.NewEffectiveVersion(pversion.Get().String())
```

Comment:
I'm not totally clear on the impact of setting this field. This isn't just used as the version number that should show to clients. See usages of the `EffectiveVersion` interface in the k/k repo. Maybe those other code paths where it is used don't effect us though?

Reply:
The `EffectiveVersion` has to be populated or it causes panics. I meant to dig more into what it really means but it definitely must be set.

Reply:
Seems okay as far as I can tell from browsing the code a little. I guess the interaction tests will tell us.

Reply:
After some investigation it looks like this field is used by the Kubernetes code to decide when to serve an API. Since Pinniped doesn't choose to serve APIs based on its own version, we could probably set any value in this field. We will choose to use the Pinniped version here so that it reads well.